app/upgrades/v0.1.8.go

```diff
@@ -0,0 +1,125 @@
+package upgrades
```

Comment:
Can you move this out into another PR as we get ready to merge this?

I fully understand why it was part of this one for testing + iteration purposes.

If you want to push back and think we should keep it - I'm open to it.

Reply:
We previously discussed that it is a good idea for the implementer of the feature to also write an upgrade code for it. Perhaps not implement the whole plan, but to add "upcoming release upgrade" code. One who implements the feature has a better understanding of the state changes necessary.

Reply:
I’d rather keep the migration logic in this PR. Writing and validating the upgrade steps while the context is still fresh makes them more reliable, and we can keep accumulating these migrations here until we schedule the release upgrade. Let me know if you see any downsides to that.